modules/builds-source-code.adoc

```diff
@@ -23,15 +23,15 @@ ifndef::openshift-online[]
   dockerfile: "FROM openshift/ruby-22-centos7\nUSER example" <3>
 endif::[]
 ----
-<1> The `git` field contains the URI to the remote Git repository of the source code. Optionally, specify the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name.
+<1> The `git` field contains the `URI` to the remote Git repository of the source code. Specify the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name, if not specified "master" will be picked as the default value.
 <2> The `contextDir` field allows you to override the default location inside the source code repository where the build looks for the application source code. If your application exists inside a sub-directory, you can override the default location (the root folder) using this field.
 ifndef::openshift-online[]
 <3> If the optional `dockerfile` field is provided, it should be a string containing a Dockerfile that overwrites any Dockerfile that may exist in the source repository.
 endif::[]
 
 If the `ref` field denotes a pull request, the system uses a `git fetch` operation and then checkout `FETCH_HEAD`.
 
-When no `ref` value is provided, {product-title} performs a shallow clone (`--depth=1`). In this case, only the files associated with the most recent commit on the default branch (typically `master`) are downloaded. This results in repositories downloading faster, but without the full commit history. To perform a full `git clone` of the default branch of a specified repository, set `ref` to the name of the default branch (for example `master`).
+When no `ref` value is provided, {product-title} performs a shallow clone (`--depth=1`). In this case, only the files associated with the most recent commit on the default branch (typically `master`) are downloaded. This results in repositories downloading faster, but without the full commit history. To perform a full `git clone` of the default branch of a specified repository, set `ref` to the name of the default branch (for example `main`).
```

Comment:
s/typically `master`/typically `main` ?

Reply:
@coreydaley could you please help with a review here? If this statement makes sense (specifically usage of master & main in the sentences). Or, we should go with updating `typically master` to `typically main`?

Reply:
Yeah, exactly. Then, it seems best to keep it as is.